### ISSUE TITLE
RSAKey from other sources

### ISSUE THREAD

**Issue by panzone** [panzone/crypto-chapel#18]
RSAKeys now can only be generated on memory, so an application can't import or export a public(or private) key. This obviously is a feature that should be provided.

The idea is to be able to create a RSAKey reading a file into some sort of format (OpenSSL uses PEM but I'm not entirely sure if it's only for certificates). We shouldn't assume that these RSAKey contains the private key. 

Also, a RSAKey should be exported to a file. In this case it's important to provide a specific procedure for exporting only the public key, even if the private is available into the keychain.

**Comment by saru95** [comment]
Absolutely, yes. This is the next on my `TODO` list. `pem` is the format that it most commonly used by most of the Crypto modules. However, it can be written to any file. There's no compulsion for the file to be something like `public.pem`.
